wallet/crate/Cargo.toml

```diff
@@ -31,10 +31,10 @@ crate-type = ["cdylib"]
 [dependencies]
 console_error_panic_hook = { version = "0.1.7", default-features = false }
 js-sys = { version = "0.3.56", default-features = false }
-manta-accounting = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.2", default-features = false, features = ["serde"] }
-manta-crypto = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.2", default-features = false, features = ["serde"] }
-manta-pay = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.2", default-features = false, features = ["groth16", "http", "scale", "serde"] }
-manta-util = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.2", default-features = false, features = ["serde"] }
+manta-accounting = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.3", default-features = false, features = ["serde"] }
+manta-crypto = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.3", default-features = false, features = ["serde"] }
+manta-pay = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.3", default-features = false, features = ["groth16", "http", "scale", "serde"] }
+manta-util = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.3", default-features = false, features = ["serde"] }
```

Comment:
@Kevingislason does this need to be downgraded to `v0.5.2`?

Reply:
yes